java/ingest/ingest-core/src/main/java/sleeper/ingest/task/IngestTaskStatus.java

```diff
@@ -24,6 +24,9 @@
 import java.time.Instant;
 import java.util.Objects;
 
+/**
+ * A class for storing the status of an ingest task.
```

Comment:
We can omit "A class" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html